app/models/customer.rb

```diff
@@ -7,4 +7,9 @@ class Customer < ApplicationRecord
   has_many :offers
   has_secure_password
   validates :password, presence: true, length: { minimum: 6 }
+
+  def customer?
+    self.role == 'customer'
+  end
```

Comment:
Czy na pewno potrzebujemy tej metody :)? Nie mamy tutaj atrybutu 'role', klient zawsze jest klientem.

Reply:
Spojrzałam, niżej i dodałaś role do tabel. Jednak tutaj nie jest to potrzebne, bo mamy rozdzielenie modelów użytkowników.